package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "unique-names-generator",
-  "version": "4.7.1",
+  "version": "4.7.2",
```

Comment:
This is not a patch but rather a new feature.
We will decide how and when to release a newer version of the package. Please, don't alter the package version in your PR